src/CTA.Rules.Metrics/PortSolutionResultReportGenerator.cs

```diff
@@ -103,13 +106,19 @@ private void GenerateMetrics()
                 actionPackageMetrics.AddRange(MetricsTransformer.TransformActionPackages(Context, projectResult));
                 actionExecutionMetrics.AddRange(MetricsTransformer.TransformGenericActionExecutions(Context, projectResult));
                 missingMetaReferenceMetrics.AddRange(MetricsTransformer.TransformMissingMetaReferences(Context, projectResult));
+
+                if (projectResult.WebFormsMetricResults != null)
+                {
+                    webFormsActionMetrics.AddRange(MetricsTransformer.TransformWebFormActionMetrics(Context, projectResult));
+                }
```

Comment:
Rather than check for a null here, it may be cleaner and safer to always initialize `WebFormsMetricResults` to an empty collection in the `ProjectResult` constructor